pkg/config/repoconfig/repoconfig.go

```diff
@@ -10,19 +10,18 @@ import (
 
 const (
 	defaultApprovals = 2
-	defaultLabel     = "ready-to-test"
+	defaultLabel     = ""
 	filename         = ".submarinerbot.yaml"
 )
 
 type BotConfig struct {
-	LabelApproved *struct {
-		Approvals *int
-		Label     *string
+	LabelApproved struct {
```

Comment:
No this should be a pointer, otherwise you might've not wanted it but even an empty file will now set defaults..

The idea was to allow to set file with
```yaml
label-approved:
```

And it'll fill the defaults of the label/reviewers.

If you want to simplify we can make those mandatory, but this field should still be optional.